store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9642,4 +9642,13 @@ TODO: delete them permanently from here
 <attr id="5003" name="zimbraCustomAppBanner" type="string" max="256" cardinality="single" optionalIn="account,cos,domain" flags="accountInfo,accountCosDomainInherited,domainAdminModifiable" since="8.8.6">
   <desc>Logo app banner URL for web client</desc>
 </attr>
+
+<attr id="5004" name="zimbraSmtpStartTlsTrustedHosts" type="string" cardinality="single" optionalIn="globalConfig,server,domain" flags="serverInherited" since="8.8.6">
+  <globalConfigValue>*</globalConfigValue>
+  <desc>Configures trusted hosts sending mail from mailboxd to MTA with starttls.
+        This parameter doesn't affect on smtp sessions to zimbraDataSourceSmtpHost.
+        *   - trust all hosts
+        host[&lt;sp&gt;host...] - A white space separated list of trusted hosts
```

Comment:
Same comments as https://github.com/Zimbra/zm-mailbox/pull/533#discussion-diff-171293237R9651
--- cut-n-paste ---
nit: change `host[&lt;sp&gt;host...] - A white space separated list of trusted hosts`
a. I believe sticking to common ~ man-ish / POSIX-ish nomenclature should suffice
b. white space → whitespace - consistent with doc https://javaee.github.io/javamail/docs/api/com/sun/mail/smtp/package-summary.html#mail.smtp.ssl.trust
so perhaps:
- `host ... - A whitespace separated list of trusted hosts` - this seems more traditional in man pages these days
OR
- `host... - A whitespace separated list of trusted hosts` - this would seem more consistent w/posix recommendations

Refs:
1. https://stackoverflow.com/questions/25962007/what-does-the-ellipsis-mean-in-man-pages
2. http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_01 item 9 / https://unix.stackexchange.com/questions/193673/usage-of-ellipse-in-synopsis-of-command-line-argument

Reply:
Corrected to conform the latter suggestion.